tests/Validation/ValidationRequiredIfTest.php

```diff
@@ -38,7 +46,7 @@ public function testItOnlyCallableAndBooleanAreAcceptableArgumentsOfTheRule()
 
         $this->expectException(\InvalidArgumentException::class);
 
-        $rule = new RequiredIf('phpinfo');
+        $rule = new RequiredIf('Taylor Otwell');
```

Comment:
This is a change in behaviour. I'm not sure if it's a good idea to send this in to 8.x

Reply:
Yeah. But I think it isn't B.C

Reply:
I think this could be a security problem when we let user passes callable from the input
```php
'name' => new RequireIf(['rmdir', '/some/directory'])
```

Reply:
It'll now invoke any string that matches a native or non-native function.

Reply:
@bangnokia It's accepted callable array, closure before. The end-user can not send this to the param.